modules/pages/client/views/faq-general.client.view.html

```diff
@@ -119,3 +119,8 @@ <h3>How can I contact you?</h3>
     <a href="https://www.facebook.com/trustroots.org/">Facebook</a>
     and to directly contact individuals, see <a ui-sref="team">the team</a> page.
 </div>
+
+<div class="faq-question" id="how-do-i-report-a-member-that-violates-the-rules">
+  <h3>How do I report a member that violates the rules?</h3>
+  If you navigate to the profile of a user you will find a link that says "Report member" that will lead you to a report form. Also, at any point you can contact support@trustroots.org with your report. Please include a link to the profile or a username if you provide the report via email.
```

Comment:
> Also, at any point you can contact support@trustroots.org with your report. 

@noahsmindfuck @rumwerfer what do you think, would it be better to just direct folks to support form instead of asking them to send an email?

```html
<a ng-href="support">support</a>
```

Reply:
yeah,sure, it's enough

Reply:
oh yeah, that would be better indeed